src/validation.cpp

```diff
@@ -3570,6 +3570,9 @@ static void FindFilesToPrune(std::set<int>& setFilesToPrune, uint64_t nPruneAfte
 
     unsigned int nLastBlockWeCanPrune = chainActive.Tip()->nHeight - MIN_BLOCKS_TO_KEEP;
     uint64_t nCurrentUsage = CalculateCurrentUsage();
+    // Worst case remaining block space:
```

Comment:
Seems like this ought to be using best case...?

Reply:
Best case would be empty blocks. That would lead to a lot of flushes.